airbyte-integrations/connectors/source-clazar/integration_tests/sample_config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
That should be removed — CATs would fetch the secrets file from the secrets manager when configured correctly. On a very low usage (when we just get started) it's okay to not have this here at all.